src/v/archival/ntp_archiver_service.cc

```diff
@@ -295,28 +295,68 @@ ss::future<cloud_storage::download_result> ntp_archiver::sync_manifest() {
         vlog(
           _rtclog.debug, "Updating the archival_meta_stm in read-replica mode");
```

Comment:
nit: would be nice to log the insync offset here to understand the recency of the manifest that we are syncing with